tox.ini

```diff
@@ -74,9 +74,7 @@ commands =
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-crypto-ethereum
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-crypto-cosmos
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-crypto-fetchai
-    pytest -rfE plugins/aea-crypto-fetchai/tests --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov=aea_crypto_fetchai --cov-config=.coveragerc --suppress-no-test-exit-code {posargs}
-    pytest -rfE plugins/aea-crypto-ethereum/tests --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov=aea_crypto_ethereum --cov-config=.coveragerc --suppress-no-test-exit-code {posargs}
-    pytest -rfE plugins/aea-crypto-cosmos/tests --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov=aea_crypto_cosmos --cov-config=.coveragerc --suppress-no-test-exit-code {posargs}
+    pytest -rfE --doctest-modules plugins/aea-crypto-fetchai/tests plugins/aea-crypto-ethereum/tests plugins/aea-crypto-cosmos/tests --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov=aea_crypto_fetchai --cov=aea_crypto_cosmos --cov=aea_crypto_ethereum --cov-config=.coveragerc --suppress-no-test-exit-code {posargs}
```

Comment:
this can't work because:

```
ImportError while loading conftest '/home/marcofavorito/workfolder/agents-aea/plugins/aea-crypto-ethereum/tests/conftest.py'.
py._path.local.LocalPath.ImportMismatchError: ('tests.conftest', '/home/marcofavorito/workfolder/agents-aea/plugins/aea-crypto-fetchai/tests/conftest.py', local('/home/marcofavorito/workfolder/agents-aea/plugins/aea-crypto-ethereum/tests/conftest.py'))
```

Reply:
`tests.conftest` name clashes across plugins.